packages/plugin-vue/src/template.ts

```diff
@@ -123,7 +123,7 @@ export function resolveTemplateCompilerOptions(
           slash(path.relative(options.root, path.dirname(filename)))
       }
     }
-  } else {
+  } else if (transformAssetUrls !== false) {
```

Comment:
If `transformAssetUrls` is `false`, should we disabled it if `options.devServer` is true too? I think we can skip most of the code around here for that case.

Reply:
Let's check this in another PR, the current branch refers to an optimization that may still be good to have for dev. I think we can merge this PR for the moment.